server/src-lib/Hasura/GraphQL/RemoteServer.hs

```diff
@@ -337,10 +338,16 @@ execRemoteGQ' env manager userInfo reqHdrs q rsi opType =  do
            , HTTP.responseTimeout = HTTP.responseTimeoutMicro (timeout * 1000000)
            }
   Tracing.tracedHttpRequest req \req' -> do
+    Tracing.attachMetadata [("field_paths", renderedFieldPaths)]
     (time, res)  <- withElapsedTime $ liftIO $ try $ HTTP.httpLbs req' manager
     resp <- either httpThrow return res
     pure (time, mkSetCookieHeaders resp, resp ^. Wreq.responseBody)
   where
+    renderFieldPath fp = T.intercalate "." $ map getFieldNameTxt (unFieldPath fp)
+    renderedFieldPaths = case fieldPaths of
+      TopLevelRemoteCall -> "toplevel"
```

Comment:
I don't think we should include this string in the metadata. Instead, I think it would be better to emit a list of root fields in the case of a top-level remote call. However, we can add that later, and for now, just not emit the `field_paths` metadata at all in this case.